test/commons/color/get-background-color.js

```diff
@@ -807,28 +802,31 @@ describe('color.getBackgroundColor', function() {
     document.body.style.height = originalHeight;
   });
 
-  (shadowSupported ? it : xit)('finds colors in shadow boundaries', function() {
-    fixture.innerHTML = '<div id="container"></div>';
-    var container = fixture.querySelector('#container');
-    var shadow = container.attachShadow({ mode: 'open' });
-    shadow.innerHTML =
-      '<div style="background-color: black;">' +
-      '<span id="shadowTarget" style="color: #ccc;">Text</span>' +
-      '</div>';
-    axe.testUtils.flatTreeSetup(fixture);
+  (shadowSupported ? it : xit)(
```

Comment:
With IE11 gone all our tests run in browsers that support shadow dom (Chrome and Firefox). We should get rid of `shadowSupported` as well